modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -698,6 +713,51 @@ Replaced by: `ALTER USER`.
 
 == Procedure descriptions
 
+[[procedure_cdc_current]]
+.cdc.current()
+[cols="<15s,<85"]
+|===
+| Description
+a|
+Fetch the current `changeId` for use with xref:reference/procedures.adoc#procedure_cdc_unfiltered_query[`cdc.unfiltered.query()`]
+
+For mor details, see link:{neo4j-docs-base-uri}/change-data-capture/{page-version}/introduction[Change Data Capture].
```

Comment:
```suggestion
For more details, see link:{neo4j-docs-base-uri}/change-data-capture/{page-version}/introduction[Change Data Capture].
```

Reply:
also applies to the other two procedures

Reply:
No longer relevant after d721ad9